keyboards/ergodox_ez/keymaps/dctucker/config.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * dctucker keymap for ergodox_ez
+ * Copyright (C) 2023 Casey Tucker <dctucker@hotmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#define PERMISSIVE_HOLD
```

Comment:
```suggestion
#pragma once

#define PERMISSIVE_HOLD
```

Reply:
Not sure where else my config.h is sourced, but sure why not.